cli/command/cli.go

```diff
@@ -171,6 +170,7 @@ func (cli *DockerCli) Initialize(opts *cliflags.ClientOptions) error {
 
 			opts.Common.TLSOptions.Passphrase = passwd
 			cli.client, err = NewAPIClientFromFlags(opts.Common, cli.configFile)
+			cli.client.NegotiateAPIVersion()
```

Comment:
Ah yes, I misunderstood the request to move this to "Initialize". I'm guessing we meant "When we remove UpdateClientVersion, then NegotiateAPIVersion should just take its place".